airflow/hooks/webhdfs_hook.py

```diff
@@ -67,9 +77,10 @@ def get_conn(self):
             except HdfsError as e:
                 self.log.debug(
                     "Read operation on namenode {nn.host} "
-                    "failed with error: {e}".format(nn=nn, e=e)
+                    "failed with error: {e}".format(**locals())
```

Comment:
Don't log all local vars, but explicitly provide the vars that you want to log. Generally speaking, explicitness is better because you know what to expect.